src/transform/ThemeTransform.js

```diff
@@ -2,11 +2,11 @@ import './ThemeTransform.css'
 import ElementUtilities from './ElementUtilities'
 import Polyfill from './Polyfill'
 
-// Elements marked with either of these classes indicate certain ancestry constraints that are
+// Elements marked with these classes indicate certain ancestry constraints that are
 // difficult to describe as CSS selectors.
 const CONSTRAINT = {
   IMAGE_NO_BACKGROUND: 'pagelib-theme-image-no-background',
-  IMAGE_NONTABULAR: 'pagelib-theme-image-nontabular'
+  DIV_DO_NOT_APPLY_BASELINE: 'pagelib-theme-div-do-not-apply-baseline'
```

Comment:
Another similar example is saying "bookmark_filled" to refer to the read more saved page icon instead of just generically "icon_enabled". What if the icon is changed to a diskette? What should just be an asset swap now becomes a project-wide scouring, possibly API breaking change.

Specificity can help in conceptually scoping down a problem to focus on writing an initial concrete version but must be immediately revised (before pushing!) to remove needless detail. The reasons are that:
- We lose a ton of flexibility to do even the most subsequent basic changes or refactoring until the code is reconsidered, reimagined, and rewritten to be malleable.
- The code reads as a one-off, custom-made implementation which discourages consideration for nearly identical problems with different details. If we make a function that sorts hot dogs but can just as easily sort cylinders, we probably shouldn't call it `sortHotDogs()`.
- The code quickly becomes outdated because many details, relevant or not, are captured and any change in the problem requires resynchronization with the code whether it was functionally important or not.
- In some cases, it's actually more difficult to read because it isn't clear why it's so specific. It'd be like asking for directions to the grocery store and someone describing every turn but also what kind of paint was used for the lines on each road. The detail is only distracting noise! Just tell me what turns to take!

An equivalent example would be needlessly naming a variable `fooList` instead of `foos` or `quicksort` instead of `sort`. When we change the list to an array, or the quicksort to a bubble sort, instead of just changing the implementation strategy (hopefully one line) we have to change _all_ references and then _struggle_ to understand if there was a specific characteristic of the lists or quicksort that was really needed or this just a case of needless specificity and there were no assumptions on the underlying collection or algorithm type. There may be some but the burden of identifying important characteristics and removing unimportant detail should be a one-time cost paid by the author, not every reader.

This all boils down to writing code that _only_ references the behavior / characteristic / interface we **need** instead of today's happenstance implementation / strategy. Another way to put it is to try to only describe what's so important, so fundamental, we can't imagine or execute our code snippet without it.